.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] As the previos reviewer mentioned kindly add some steps on how to use the project in `Geting started` section on your README
e.g: 
```
first, you need to clone the project
- git clone URL.git
second enter the folder
- cd urlProject
```